napari/plugins/_tests/test_hook_specifications.py

```diff
@@ -80,18 +79,3 @@ def test_docs_match_signature(name, func):
         "not match the parameters listed in the docstring:\n"
         f"{sig_params} != {doc_params}"
     )
-
-    # we know the parameters names match, now check that their types match...
-    # but only emit a warning if not
-    for doc_param in docs.get('Parameters'):
-        sig_param = sig.parameters.get(doc_param.name)
-        name = getattr(sig_param.annotation, '_name', None)
-        name = name or getattr(sig_param.annotation, '__name__', None)
-        if doc_param.type != name:
-            msg = (
-                f'The type ({name}) for parameter '
-                f'"{sig_param.name}" in hook specification "{name}" does not '
-                'match the type specified in the docstring '
-                f'({doc_param.type})'
-            )
-            warnings.warn(msg)
```

Comment:
there wasn't any discussion.  it was just error prone and led to lots of warnings in the tests.  The goal was to make sure that docstrings matched type annotations, but since there's no standard for docstring syntax, it's _very_ hard to test equality... and it almost always warns that they are different.